cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2387,6 +2387,47 @@
     </description>
   </property>
 
+  <property>
+    <name>app.program.runtime.monitor.retry.policy.base.delay.ms</name>
```

Comment:
The runtime monitor is a system component, use `system.runtime.monitor.` as the prefix instead. Also you need to update the MD5 hash for the doc.